### PR TITLE
Make it possilbe to remove articles from group

### PR DESCRIPTION
**Pull request by tu55eladd**

### CODE REVIEW

**Comment by tu55eladd** [comment]
Branch is based add-article branch and might seem big but is really just last four commits.

**Comment by duvholt** [comment]
Should we close the other pull request and do review on this one instead? Or finish review on the other pr and then this?

**Comment by tu55eladd** [comment]
Think the other one should be merged first, then this one. Hopefully "duplicated commits" will disappear when the other one is merged.

File: src/reducers/articleGroups.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const articleGroup = (state = {}, action) => {
       return state;
     }
 
-    default:
+        default:
```
Comment (duvholt) [inline src/reducers/articleGroups.js]: Accident indent I guess?

File: src/reducers/articleGroups.js [hunk 2]
```diff
@@ -85,6 +86,15 @@ const articleGroups = (state = {}, action) => {
       });
     }
 
+    case DELETE_ARTICLE_FROM_GROUP: {
+      const { articleId, groupId } = action.payload;
+      const articleIndex = state[groupId].articles.indexOf(articleId);
+
+      state = Object.assign({}, state);
+      state[groupId].articles.splice(articleIndex, 1);
```
Comment (duvholt) [inline src/reducers/articleGroups.js]: This is mutating the articles list. Meaning that the previous state is also changed.

Use the `removeFromArticles` helper function.
Also I think it's cleaner to do this in the subreducer I made. Call it like `ADD_ARTICLE_TO_GROUP` does.

**Review by duvholt — CHANGES_REQUESTED** [review]

**Comment by tu55eladd** [comment]
Should be good now

**Review by duvholt — APPROVED** [review]

**Pull request merged**